users/migrations/0022_follow_request.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 4.2.3 on 2023-08-04 01:38
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("users", "0021_identity_aliases"),
+    ]
+
+    operations = [
+        migrations.RunSQL("DELETE FROM users_follow WHERE state <> 'accepted';"),
```

Comment:
This isn't safe - you'll nuke all in-progress follow handshakes that are going on. What are you trying to achieve here?

Reply:
I want to ensure that any follows that are in-flight at the time of the migration (i.e. in the `local_requested` or `remote_requested` state) will still follow through and finish.